UnitsNet/GeneratedCode/Quantities/Ratio.g.cs

```diff
@@ -97,25 +97,6 @@ public Ratio(double value, RatioUnit unit)
             _unit = unit;
         }
 
-        /// <summary>
-        /// Creates an instance of the quantity with the given numeric value in units compatible with the given <see cref="UnitSystem"/>.
-        /// If multiple compatible units were found, the first match is used.
-        /// </summary>
-        /// <param name="value">The numeric value to construct this quantity with.</param>
-        /// <param name="unitSystem">The unit system to create the quantity with.</param>
-        /// <exception cref="ArgumentNullException">The given <see cref="UnitSystem"/> is null.</exception>
-        /// <exception cref="ArgumentException">No unit was found for the given <see cref="UnitSystem"/>.</exception>
-        public Ratio(double value, UnitSystem unitSystem)
-        {
-            if (unitSystem is null) throw new ArgumentNullException(nameof(unitSystem));
-
-            var unitInfos = Info.GetUnitInfosFor(unitSystem.BaseUnits);
-            var firstUnitInfo = unitInfos.FirstOrDefault();
-
-            _value = value;
-            _unit = firstUnitInfo?.Value ?? throw new ArgumentException("No units were found for the given UnitSystem.", nameof(unitSystem));
-        }
-
```

Comment:
I'm fine removing this ctor as I hardly think anyone uses it, but would it not also be equally valid to let the ctor construct a ratio with unit `BaseUnit` (`DecimalFraction`)? 

It seems consistent with behavior that `As(UnitSystem)` also uses `BaseUnit`, and quantity implementations would be more consistent with each other.

Reply:
I think that both constructors should go, but since this would have been a _sudden_ breaking change I kept the one that could have actually be used by someone..

Reply:
Alright